builds/e2e/connectivity.yaml

```diff
@@ -4,6 +4,7 @@ pr: none
 variables:
   images.artifact.name.linux: 'core-linux'
   vsts.project: $(System.TeamProjectId)
+  NugetSecurityAnalysisWarningLevel: warn
```

Comment:
?

Reply:
I added to be able to run the connectivity tests, it was failing to run the pipeline otherwise.